src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt

```diff
@@ -46,7 +46,7 @@ import org.jf.dexlib2.iface.reference.StringReference
     ]
 )
 @Name("sponsorblock")
-@Description("Integrate SponsorBlock.")
+@Description("Integrate SponsorBlock in video player to skip sponsorship segments in YouTube videos.")
```

Comment:
```suggestion
@Description("Integrates SponsorBlock which allows skipping video segments such as sponsored content.")
```

Reply:
Resolved in https://github.com/revanced/revanced-patches/pull/1571/commits/880014fa05c2d223c0827079ae9279828858e048#diff-2d0b9f662f2e83de95f468244a0db136b6543c779e90efa21d346db4f19e4483